### DISCUSSION TITLE
How can we prevent new plots from starting?

### DISCUSSION THREAD

**Discussion by cdeniz** [swar/Swar-Chia-Plot-Manager#442] [Q&A]
For example, 5 plots are running in parallel. I want to finish these plots and I don't want that the new plots to start. 
How can I set it up during manager running?

**Comment by RaySmalley** [comment]
I'm running into the issue where 5 or more plot jobs are starting (and filling up my temp dirs) when the config.yaml clearly states:

    max_concurrent: 3
    max_concurrent_with_start_early: 4
    stagger_minutes: 90
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Comment by RaySmalley** [comment]
Actually, I found out that my temp dirs on 2 of my plotters were full. Is there any logic built in to not start new plots if this happens?

Edit: Even after stopping a stuck job and clearing out its temp files, Swar is still running more than 4 jobs at a time. This is not good.

**Comment by dJani97** [comment]
If you want your plots to finish and not start any new ones, use `python manager.py stop`

Do you reload your config after changing it using `python manager.py restart`?

None of the above commands will kill your current plots.

**Comment by RaySmalley** [comment]
I figured out my problem, and I apologize to OP for hijacking his post when it turned out to be unrelated. In my case I had renamed my job (from the default micron) midway through, and apparently swar doesn't count the old jobs towards the max concurrent jobs.
